script/DeployInput.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.20;
 
 contract DeployInput {
-  uint256 constant INITIAL_VOTING_DELAY = 7200; // 24 hours
+  uint256 constant INITIAL_VOTING_DELAY = 3600; // 12 hours
```

Comment:
is the clock mode on this contract blocks or timestamp? in either case, is this correct?

Reply:
Blocks... the RadworksGovernor inherits from GovernorVotesComp, and GovernorVotesComp defines its `clock` function in such as way that if the underlying governance token has a `clock` function, that it called and its value is returned.. if it does not (and the Radicle token does not) the current block number is returned:

```
(from GovernorVotesComp)
   /**
     * @dev Clock (as specified in EIP-6372) is set to match the token's clock. Fallback to block numbers if the token
     * does not implement EIP-6372.
     */
    function clock() public view virtual override returns (uint48) {
        try token.clock() returns (uint48 timepoint) {
            return timepoint;
        } catch {
            return SafeCast.toUint48(block.number);
        }
    }
```
At 12 seconds per block (5 per minute) an hour's worth of blocks would be 300 .. so 12 hours would be 3600.

Reply:
thanks for the explanation @jferas, i stand corrected, looks good to me!